web/packages/teleterm/src/ui/Tabs/TabItem.tsx

```diff
@@ -137,7 +137,12 @@ const RelativeContainer = styled.div`
   height: 100%;
 `;
 
-const TabContent = styled.div`
+const TabContent = styled.div<{
+  dragging?: boolean;
+  active?: boolean;
+  // TODO(bl-nero): is this really used? Perhaps remove it.
+  canDrag?: boolean;
```

Comment:
Yes, I believe it is used when there are no active tabs.

https://github.com/gravitational/teleport/blob/0fd5b0c29b7a3119644cf1856ddbaade96a59dfd/web/packages/teleterm/src/ui/Tabs/Tabs.tsx#L62-L64

https://github.com/gravitational/teleport/blob/0fd5b0c29b7a3119644cf1856ddbaade96a59dfd/web/packages/teleterm/src/ui/Tabs/TabItem.tsx#L55

https://github.com/gravitational/teleport/blob/0fd5b0c29b7a3119644cf1856ddbaade96a59dfd/web/packages/teleterm/src/ui/Tabs/TabItem.tsx#L77-L81

Reply:
Oh, I see. I just looked it up and it can be removed from `TabContent`. It used to be used for [selectively applying `cursor: pointer`](https://github.com/gravitational/teleport/blob/d4f5f2fce37ef41df478845f0d92bc30243eaacf/web/packages/teleterm/src/ui/Tabs/TabItem.tsx#L103-L105), but this code has since been removed. It still has to be passed to the `useTabDnD` hook though.